fluent-package/make-index-html.rb

```diff
@@ -0,0 +1,49 @@
+require 'erb'
+require 'find'
+require 'pathname'
+require 'optparse'
+
+options = {
+  :site => "https://fluentd.cdn.cncf.io",
+  :verbose => false
+}
+
+opt = OptionParser.new
+opt.on("-s", "--site [URL]") { |v| options[:site] = v }
+opt.on("-v", "--verbose") { |v|
+  options[:verbose] = v
+}
+top_dir = opt.parse!(ARGV).first
+
+unless File.exist?(top_dir)
+  puts "#{top_dir} not found"
+  exit 1
+end
+
+template_path = File.expand_path("#{File.basename(__FILE__, ".rb")}.erb")
+puts "Template path: #{template_path}"
+%w(5 lts test).each do |channel|
+  search_path = Pathname.new("#{top_dir}/#{channel}")
+  Find.find(search_path).each do |path|
+    Find.prune if path.end_with?(".deb")
+    Find.prune if path.end_with?(".msi")
+    Find.prune if path.end_with?(".gz")
+    Find.prune if path.end_with?(".bz2")
+    Find.prune if path.end_with?(".rpm")
```

Comment:
Is there any reason to check suffix?
It seems that we can skip all regular files:
```suggestion
    Find.prune unless FileTest.directory?(path)
```

Reply:
Thanks!